app/views/products/index.html.erb

```diff
@@ -51,6 +51,54 @@
     <% end %>
   </div>
   <div class="product-title">
-    <h2>Woman's</h2>
+    <h2>Woman</h2>
+  </div>
+  <div class="product page-flex">
+    <% @products.each do |product| %>
```

Comment:
Performance not good (looping the same array n time). Suggestion:
- Use `group` or `group_by` to group product by categories
- Use partial view